src/main/resources/edu.umd.lib.wufoo-connector-camel.cfg

```diff
@@ -8,11 +8,14 @@ wufoo.routeName=wufoo
 wufoo.serviceName=wufoo-listener
 wufoo.handshake_key=testing
 
+# Wufoo Sysaid Field Mapping 
+wufoo.Ramesh\u0020Test\u0020-\u0020DSS.mapping=Wufoo-Sysaid-Mapping.properties
+
 #Properties required for SysAid Connection 
 SysAid.serviceName=wufoo-listener
 SysAid.url=https://libticketingdev.umd.edu/api/v1/
-SysAid.userName=********
-SysAid.password=*********
+SysAid.userName=*******
+SysAid.password=*******
```

Comment:
Change the configuration to make all the keys to be in lowercase (e.g sysaid.servicename). Also, leave the values blank for keys that do not have default values (the '*'es can be removed).